taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/exceptions/SystemException.java

```diff
@@ -0,0 +1,15 @@
+package pro.taskana.adapter.camunda.outbox.rest.exceptions;
```

Comment:
Why do we need this exception here in the adapter? Can't the adapter just throw a RuntimeException? I don't like having the same exception here and in taskana-core

Reply:
It sure could. Introduced it to be consistent with core, but I can change it